var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -48,6 +48,9 @@ class Hypre(Package):
             description="Use internal Superlu routines")
     variant('int64', default=False,
             description="Use 64bit integers")
+    
+    # Temporary patch to add ppc64le in config.guess
+    patch('ibm-ppc64le.patch')
```

Comment:
more precisely,
```
patch('ibm-ppc64le.patch', when='@:2.11.1')
```
and then also remove 

> Temporary

because the patch will stay fold old versions.